app/scripts/controllers/exportImport.js

```diff
@@ -126,6 +126,12 @@ angular.module('openhimConsoleApp')
       var exportData = angular.copy( $scope.selectedExports );
       var textFile = null;
 
+      // OHM-382: warn when exporting key
+      Alerting.AlertReset();
```

Comment:
if the user exported keystore inclusive, the alert would be posted but without reloading page if they re-eported excluding keystore i needed to remove the alert that's why i put it out!! do you still want it enclosed within if statement?